sso_server/email_template/passwordRecovery.html

```diff
@@ -0,0 +1,160 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml" xmlns:o="urn:schemas-microsoft-com:office:office" style="width:100%;font-family:helvetica, 'helvetica neue', arial, verdana, sans-serif;-webkit-text-size-adjust:100%;-ms-text-size-adjust:100%;padding:0;Margin:0">
+ <head>
+  <meta charset="UTF-8">
+  <meta content="width=device-width, initial-scale=1" name="viewport">
+  <meta name="x-apple-disable-message-reformatting">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta content="telephone=no" name="format-detection">
+  <title>Nouveau modèle de courrier électronique 2020-07-17</title>
```

Comment:
?

Reply:
Just the html template for the passwordrecovery email

Reply:
Sorry, I was hinting at the title which does not seem relevant

Reply:
password_recovery_email_template.html ?

Reply:
`<title>Nouveau modèle de courrier électronique 2020-07-17</title>`